lib/awful/widget/prompt.lua

```diff
@@ -46,10 +46,11 @@ local function run(promptbox)
     }
 end
 
-local function spawn_and_handle_error(self, ...)
+local function spawn_and_handle_error(_, ...)
```

Comment:
mb better turn it from local function to `prompt.spawn_and_handle_error`?
so other users would be able to redefine the error handler behavior (if they want, let's say, to return the old behavior)